packages/fuels/src/cli/commands/deploy/proxy/contract/fuel-toolchain.toml

```
@@ -0,0 +1,6 @@
+[toolchain]
+channel = "testnet-2024-09-06"
+
+[components]
+forc = "0.63.5"
+fuel-core = "0.35.0"
```

```suggestion
fuel-core = "0.36.0"
```
Should we be using latest [v0.36.0](https://github.com/FuelLabs/fuel-core/releases/tag/v0.36.0)